settings/installer/ospd/ospd.spec

```diff
@@ -0,0 +1,119 @@
+---
+subparsers:
+    ospd:
+        help: Installs openstack using OSP Director
+        groups:
+            - title: Firewall
+              options:
+                firewall-rules:
+                    type: YamlFile
+                    help: The list of firewall rules
+                    default: default.yml
+
+            - title: Product
+              options:
+                product:
+                    type: YamlFile
+                    help: The product to install
+                    default: "rhos-latest.yml"
+
+            - title: Undercloud
+              options:
+                undercloud-network:
+                    type: YamlFile
+                    help: The undercloud network config
+                    default: default.yml
+
+            - title: Overcloud
```

Comment:
What I would like to see instead of "undercloud" and "overcloud" groups is installer oriented groups.
So, use "images" as a group so we won't need --overcloud-images (for example) 
use "network" group, etc..